shell-ui/src/alerts/alertHooks.js

```diff
@@ -6,7 +6,7 @@ import type { FilterLabels, Alert } from './services/alertUtils';
 import { getHealthStatus, filterAlerts } from './services/alertUtils';
 
 export const getPlatformAlertSelectors = (): FilterLabels => {
-  return { alertname: ['PlatformAtRisk', 'PlatformDegraded'] };
+  return { alertname: ['ClusterAtRisk', 'ClusterDegraded'] };
```

Comment:
I'm wondering wether we should not include both the old alert names and the new ones in this filter. Because when upgrading we could loose historical alerts in the global health component. But maybe this is a none existing issue if no one is using those alerts in prod yet.

Reply:
This lib hasn't been released yet, right? So I'm not sure how upgrade is a concern.
Some deployments may have alert rules matching the old names in place, and we may not remove them, but the lib won't care and won't fetch them.

Reply:
The lib is not yet released indeed, however we already have packaged those alerts in product that are released and my concern is that for such installations an upgrade to Metalk8s 2.11 bundling the `dashboard page` would reflect a wrong history of alerts (pre upgrade some `PlatformAtRisk` or `PlatformDegraded` could be raised and then post upgrade we will watch `ClusterAtRisk` and `ClusterDegraded` which will then mislead the user experience

Reply:
Mmh, I consider it a feature addition, the previous alerts were not part of MetalK8s offering, so I don't think we should care.